website/pages/blog/nextra-4/index.mdx

```diff
@@ -0,0 +1,1228 @@
+[Refreshing the Next.js App Router When Your Markdown Content Changes](https://steveruiz.me/posts/nextjs-refresh-content)
+where you need to install unnecessary dependency like `concurrently` and `ws`, mentioned article is
+adapted on
+[Dan Abramov workaround with `<AutoRefresh>` component and dev web socket server](https://github.com/hashicorp/next-remote-watch/issues/42#issuecomment-1794052655).
+Nextra's `content` mode provide ultimate DX:
+
+- you don't need to install unnecessary dependencies
+- you don't need to restart your server on changes in `content` directory
+- hot reloading works out of box
+- you can use `import` statements in MDX files and
+  [static images](https://nextra.site/docs/guide/image#static-image) works as well
+
+See Nextra's
+[example of docs website](https://github.com/shuding/nextra/tree/d5cb1bc0e84494cbcc8decbae56fb091ce9999f0/examples/docs)
+and
+[i18n website example](https://github.com/shuding/nextra/tree/d5cb1bc0e84494cbcc8decbae56fb091ce9999f0/examples/swr-site)
+as an examples of this mode.
+
+</Callout>
+
+</Steps>
+
+#### `page` File Convention
+
+The second mode follows
+[Next.js App Router convention](https://nextjs.org/docs/getting-started/project-structure#app-routing-conventions)
+but with enhanced page extensions as `page.{md,mdx}` and don't requires additional steps to setup.
+
+Above [`content` directory convention structure](#add-mdxpathpagejsx-file), with this mode will
+looks like:
+
+<FileTree>
+  <FileTree.Folder name="app" open>
+    <FileTree.File name="layout.jsx" />
+    <FileTree.File name="page.jsx" active />
+    <FileTree.Folder name="docs" open>
+      <FileTree.File name="page.mdx" active />
+      <FileTree.Folder name="getting-started" open>
+        <FileTree.File name="page.mdx" active />
+      </FileTree.Folder>
+    </FileTree.Folder>
+  </FileTree.Folder>
+</FileTree>
+
+<Callout>
+**Warning**
+
+All pages `page.{jsx,tsx}` should export
+[`metadata` object](https://nextjs.org/docs/app/building-your-application/optimizing/metadata#static-metadata).
+
+</Callout>
+
+<Callout type="info">
+**Note**
+
+See
+[Nextra website](https://github.com/shuding/nextra/tree/d5cb1bc0e84494cbcc8decbae56fb091ce9999f0/docs)
+or Nextra's
+[blog example](https://github.com/shuding/nextra/tree/d5cb1bc0e84494cbcc8decbae56fb091ce9999f0/examples/blog)
+as an examples of this mode.
+
+</Callout>
+
+## Turbopack Support
+
+After [more than 2 years of requesting this feature](https://github.com/shuding/nextra/issues/1054),
+Nextra 4 supports [Turbopack](https://nextjs.org/docs/architecture/turbopack), incremental bundler
+written in Rust. To benefit it, simple append the `--turbopack` flag to your development command:
+
+```diff filename="package.json"
+"scripts": {
+-  "dev": "next dev"
++  "dev": "next dev --turbopack"
+}
+```
+
+<Callout type="info">
+**Note**
+
+Without `--turbopack` flag Next.js under the hood uses
+[Webpack](https://github.com/webpack/webpack), written in JavaScript.
+
+</Callout>
```

Comment:
I don't think it is really necessary. I have seen turbopack have some weird issues so would say webpack way is still the most stable option.   
```suggestion
```

Reply:
Why? Just tip for users that without turbopack - JavaScript powered Webpack is used 😅